.github/workflows/test.yml

```diff
@@ -112,11 +115,22 @@ jobs:
       - name: Configure AWS credentials
         uses: aws-actions/configure-aws-credentials@v1.6.1
         with:
-          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws-region: ap-southeast-2
           mask-aws-account-id: true
-          role-to-assume: ${{ secrets.AWS_ASSUME_ROLE_CI }}
+          role-to-assume: ${{ secrets.AWS_ASSUME_ROLE_CI_OIDC }}
```

Comment:
Role names should not be secrets. We should probably hard-code this to make it easier to debug role issues.